use-cases/using-Intel-QAT-and-kata.md

```diff
@@ -0,0 +1,491 @@
+    + [Allow insecure modules to load to host](#allow-insecure-modules-to-load-to-host)
+    + [Download Intel QAT drivers](#download-intel-qat-drivers)
+    + [Compile Intel QAT drivers against host kernel](#compile-intel-qat-drivers-against-host-kernel)
+    + [Copy Intel QAT configuration files and enable VF’s](#copy-intel-qat-configuration-files-and-enable-vf-s)
+    + [Expose and Bind Intel QAT virtual functions to VFIO-PCI (Every reboot)](#expose-and-bind-intel-qat-virtual-functions-to-VFIO-pci--every-reboot-)
+    + [Check Intel QAT virtual functions are enabled](#check-intel-qat-virtual-functions-are-enabled)
+  * [Prepare Kata Containers](#prepare-kata-containers)
+    + [Download Kata Kernel Source](#download-kata-kernel-source)
+    + [Build Kata Kernel](#build-kata-kernel)
+    + [Copy Kata Kernel](#copy-kata-kernel)
+    + [Prepare Kata root filesystem](#prepare-kata-root-filesystem)
+    + [Compile Intel QAT drivers for Kata Containers Kernel and add to Kata Containers rootfs](#compile-intel-qat-drivers-for-kata-containers-kernel-and-add-to-kata-containers-rootfs)
+    + [Copy Kata rootfs](#copy-kata-rootfs)
+    + [Update Kata configuration.toml to point to custom kernel and rootfs](#update-kata-configurationtoml-to-point-to-custom-kernel-and-rootfs)
+  * [Verify Intel QAT works in a Docker Kata Containers container](#verify-intel-qat-works-in-a-docker-kata-containers-container)
+  * [Build OpenSSL Intel QAT engine container](#build-openssl-intel-qat-engine-container)
+    + [Test Intel QAT in Docker](#test-intel-qat-in-docker)
+    + [Troubleshooting](#troubleshooting)
+  * [Optional Scripts](#optional-scripts)
+    + [Verify Intel QAT card counters are incremented](#verify-intel-qat-card-counters-are-incremented)
+    + [Unbind all VF’s from VFIO driver, reset, and rebind to the host](#unbind-all-vf-s-from-VFIO-driver--reset--and-rebind-to-the-host)
+
+# Introduction
+
+Intel\* QuickAssist Technology (Intel\* QAT) provides hardware acceleration 
+for security (cryptography) and compression. These instructions cover the 
+steps for [Clear Linux](https://clearlinux.org/)\* but can be adapted to any 
+Linux\* distribution. Your distribution may already have the Intel QAT 
+drivers, but it is likely they are do not contain the necessary user space 
+components. These instructions guide the user on how to download the kernel 
+sources, compile kernel driver modules against those sources, and load them 
+onto the host as well as preparing a specially built Kata Containers® kernel 
+and custom Kata Containers rootfs.  
+
+## Helpful Links before starting
+
+[Intel QAT Engine](https://github.com/intel/QAT_Engine)
+
+[Intel QuickAssist Technology at 01.org](https://01.org/
+intel-quickassist-technology)
+
+[hIntel Device Plugin for Kubernetes](https://github
+.com/intel/intel-device-plugins-for-kubernetes/)
+
+[Intel QuickAssist Crypto Poll Mode Driver](https://dpdk-docs.readthedocs.io/
```

Comment:
This URL is still folded (and thus broken).

Reply:
Will update. I just had not finished fully validating the instructions again with all my changes because of slow download. I'll let you know when I resubmit.